WordPress/Sniffs/Files/FileNameSniff.php

```diff
@@ -127,7 +127,7 @@ public function register() {
 	 */
 	public function process_token( $stackPtr ) {
 
-		$file     = $this->phpcsFile->getFileName();
+		$file     = trim( $this->phpcsFile->getFileName(), "\x22\x27" );
```

Comment:
WPCS actually contains a utility function to do quote stripping properly & consistently, you can call it using `$this->strip_quotes( $file )`.

https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/develop/WordPress/Sniff.php#L865-L867